h2o-py/h2o/estimators/stackedensemble.py

```diff
@@ -239,3 +239,18 @@ def metalearner_params(self, metalearner_params):
             self._parms["metalearner_params"] = None
 
 
+    @property
+    def seed(self):
+        """
+        Seed for metalearner algorithm
```

Comment:
Let's change this to: `"Seed for random numbers; passed through to the metalearner algorithm. Defaults to -1 (time-based random number)"`